tests/regression/home/gold/clausalmods-basque/result

```diff
@@ -1,15 +1,15 @@
+1@0@-1@-1@-1@-1@-1@-1@-1@-1@(376 adj-head-scop 0.000000 0 6 (367 when-s-attach-modifying-clause-phrase 0.000000 0 3 (366 decl-head-opt-subj 0.000000 0 3 (365 comp-head 0.000000 0 3 (360 bare-np 0.000000 0 1 (359 absolutive-lex 0.000000 0 1 (7 lana 0.000000 0 1 ("lana")))) (364 comp-head 0.000000 1 3 (361 imp-suffix 0.000000 1 2 (8 bukatz 0.000000 1 2 ("bukatzen"))) (363 you-agr-lex 0.000000 2 3 (362 ela-suffix1 0.000000 2 3 (9 duz 0.000000 2 3 ("duzela")))))))) (375 decl-head-opt-subj 0.000000 3 6 (374 comp-head 0.000000 3 6 (369 bare-np 0.000000 3 4 (368 allative-suffix 0.000000 3 4 (10 afaltze 0.000000 3 4 ("afaltzerat")))) (373 comp-head 0.000000 4 6 (370 fut-suffix 0.000000 4 5 (11 joa 0.000000 4 5 ("joango"))) (372 we-agr-lex 0.000000 5 6 (371 matrix-lex 0.000000 5 6 (12 gara 0.000000 5 6 ("gara"))))))))@@("S" ("PP" ("S" ("VP" ("NP" ("N" ("N" ("lana")))) ("V" ("V" ("V" ("bukatzen"))) ("V" ("V" ("V" ("duzela")))))))) ("S" ("VP" ("NP" ("N" ("N" ("afaltzerat")))) ("V" ("V" ("V" ("joango"))) ("V" ("V" ("V" ("gara"))))))))@[ LTOP: h0 INDEX: e2 [ e SF: prop-or-ques E.TENSE: tense E.ASPECT: aspect E.MOOD: mood ] RELS: < [ "_when_subord_rel"<-1:-1> LBL: h4 ARG0: i5 ARG1: h6 ARG2: h7 ]  [ "exist_q_rel"<-1:-1> LBL: h8 ARG0: x9 [ x SPECI: bool COG-ST: cog-st PNG.PER: person PNG.NUM: number ] RSTR: h10 BODY: h11 ]  [ "_job_n_rel"<-1:-1> LBL: h12 ARG0: x9 ]  [ "_finish_v_rel"<-1:-1> LBL: h13 ARG0: e14 [ e SF: prop-or-ques E.TENSE: tense E.ASPECT: imperfective E.MOOD: mood ] ARG1: x15 [ x SPECI: bool COG-ST: in-foc PNG.PER: 2nd PNG.NUM: singular ] ARG2: x9 ]  [ "exist_q_rel"<-1:-1> LBL: h16 ARG0: x17 [ x SPECI: bool COG-ST: cog-st PNG.PER: person PNG.NUM: number ] RSTR: h18 BODY: h19 ]  [ "_dinner_n_rel"<-1:-1> LBL: h20 ARG0: x17 ]  [ "_go_v_rel"<-1:-1> LBL: h21 ARG0: e2 ARG1: x22 [ x SPECI: bool COG-ST: in-foc PNG.PER: 1st PNG.NUM: plural ] ARG2: x17 ] > HCONS: < h0 qeq h1 h6 qeq h21 h7 qeq h13 h10 qeq h12 h18 qeq h20 > ICONS: < e23 [ e SF: iforce E.TENSE: tense E.ASPECT: aspect E.MOOD: mood ] non-focus x15 e24 [ e SF: iforce E.TENSE: tense E.ASPECT: aspect E.MOOD: mood ] non-focus x22 > ]@((:ascore . 0.0) (:probability . 1.0))
```

Comment:
It seems concerning that these ICONS have relations between an event not referenced anywhere else in the MRS and another event.

Reply:
I will look!

Reply:
I will open an issue for this one. I don't immediately know how to fix this, and it is unrelated to appends.

Reply:
Thanks for pointing this out, TJ!

Reply:
to clarify: this may be related to appends generally but it is unrelated to replacing diff-lists with append-lists. We used to not get ICONS at all for these ones, now we get some ICONS, not yet correct ones, but fixing that is a task specific for the clausal modifiers library.